jung-algorithms/src/main/java/edu/uci/ics/jung/layout/model/AbstractLayoutModel.java

```diff
@@ -59,6 +59,10 @@ public void stopRelaxer() {
     setRelaxing(false);
   }
 
+  public CompletableFuture getTheFuture() {
+    return theFuture;
+  }
+
   /**
```

Comment:
I put back this getter. It was in earlier versions, and I decided that users (like me in a unit test) may like to be able to join or run after relaxer thread.

Reply:
Why not put this signature in IterativeLayoutAlgorithm?  
(It's very rare that I'd rather have a public method defined in an abstract class rather than in the interface it's implementing.)

Reply:
The member is in the model, not in the algorithm. I introduced the CompletableFuture in the AbstractLayoutModel, but did not want to make that the 'only' way to run the relax thread.